inc/notificationtargetnotification.class.php

```diff
@@ -141,16 +141,19 @@ private function buildPDF($user_name = '')
             $image_width  = imagesx($image);
             $image_height = imagesy($image);
 
-            $format = '%e';
```

Comment:
I'm not entirely sure it's OK; it's not really easy to review. Seems correct at first view.

Reply:
`strftime()` uses the system localization (via the locale set with `setlocale()`), so month and day names are displayed in the configured language (e.g., “mars” in French).

`DateTime::format()` always outputs month and day names in English for formats like `F` (full month), `M` (short month), `l` (full day), etc., without localization.

Therefore, switching to `DateTime::format()` in the code you showed will always display month names in English, even if the locale is set to French.